### PR TITLE
Add command to show network ids

### PR DESCRIPTION
**Pull request by mariano-aguero**
Closes issue #17 

Examples of how to tested:

```
$ eth  networks ids
$ eth  networks ids --display=table
$ eth  networks ids --display=json
$ eth  networks --help
```

Kind regards

### CODE REVIEW

**Comment by fvictorio** [comment]
I think `eth network ids` had won the People's Choice.

File: src/networks.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module.exports.getIds = () => {
```
Comment (fvictorio) [inline src/networks.js]: Two things:

1. Maybe we can also include the urls here (right now they are in `index.js`)
2. I'm not sure if this needs to be a function. Just exporting the object should be enough.

Comment (mariano-aguero) [reply src/networks.js]: @fvictorio 

> Maybe we can also include the urls here (right now they are in index.js)

Refactor done

> I'm not sure if this needs to be a function. Just exporting the object should be enough.

Exported the ids and the urls

Note:
Added a subcommand to show the urls

**Comment by mariano-aguero** [comment]
Added new subcomand

```
$ eth  networks urls
$ eth  networks urls --display=table
$ eth  networks urls --display=json
$ eth  networks --help
```

**Comment by mariano-aguero** [comment]
@fvictorio 

More changes added:

- Change string 'SubCommand' to 'Subcommand'
- Show help when using the command without subcommand `$ eth network`

See images
screenshot_20180906_233734
screenshot_20180906_233705

**Pull request merged**